src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,135 @@
+        printWithLine(List.of("Hello! I'm Duke", "What can I do for you?"));
+
+        String line;
+        Scanner in = new Scanner(System.in);
+        List<Task> item = new ArrayList<>();
+
+        do{
+            line = in.nextLine();
+            String keyword = line.split(" ")[0].toLowerCase();
+
+            try {
+                checkLineEmpty(line);
+                checkWord(line);
+                checkDescription(line);
+
+                if (line.equalsIgnoreCase("list")) {
+                    printWithLine(List.of());
+                    List<String> messages = new ArrayList<>();
+                    System.out.println("   Here are the tasks in your list: ");
+                    for (int i = 0; i < item.size(); i++) {
+                        messages.add(i + 1 + "." + item.get(i));
+                    }
+                    printWithLine(messages);
+
+                } else if (line.split(" ")[0].equalsIgnoreCase("done")) {
+                    printWithLine(List.of());
+                    Task markItem = item.get(Integer.parseInt(line.substring(5)) - 1);
+                    markItem.markAsDone();
+                    printWithLine(List.of("Nice! I've marked this task as done: ", " " + markItem));
+
+                } else if (line.split(" ")[0].equalsIgnoreCase("todo")) {
+                    printWithLine(List.of());
+                    Task todoTask = new Todo(line.replace(line.split(" ")[0] + " ", ""));
+                    item.add(todoTask);
+                    printWithLine((List.of("Got it. I've added this task: ", todoTask.toString(), "Now you have " + item.size() + " task in the list. ")));
+
+                } else if (line.split(" ")[0].equalsIgnoreCase("deadline")) {
+                    printWithLine(List.of());
+                    int position = line.indexOf("/");
+                    String time = line.split("/")[1].replace("by ", "");
+                    Task deadlineTask = new Deadline(line.substring(9, position - 1), time);
+                    item.add(deadlineTask);
+                    printWithLine((List.of("Got it. I've added this task: ", deadlineTask.toString(), "Now you have " + item.size() + " task in the list. ")));
+
+                } else if (line.split(" ")[0].equalsIgnoreCase("event")) {
+                    printWithLine(List.of());
+                    int position = line.indexOf("/");
+                    String time = line.split("/")[1].replace("at ", "");
+                    Task eventTask = new Event(line.substring(6, position - 1), time);
+                    item.add(eventTask);
+                    printWithLine((List.of("Got it. I've added this task: ", eventTask.toString(), "Now you have " + item.size() + " task in the list. ")));
+
+                } else if (line.equalsIgnoreCase("bye")) {
+                    printWithLine(List.of());
+                    printWithLine(List.of("Bye. Hope to see you again soon!"));
+
+                }
+            }
```

Comment:
Small Feedback, perhaps utilizing a design pattern instead of using multiple if-else statements would improve readability?